common/src/main/java/net/william278/papiproxybridge/api/PlaceholderAPI.java

```diff
@@ -90,7 +87,16 @@ public static PlaceholderAPI getInstance() {
      */
     @ApiStatus.Internal
     public static void register(@NotNull PAPIProxyBridge plugin) {
-        instance = new PlaceholderAPI(plugin);
+        PlaceholderAPI.plugin = plugin;
+        instance = new PlaceholderAPI();
+    }
+
+    /**
+     * Create a new instance of PlaceholderAPI allowing unique customisation of caching mechanisms
+     * @return PlaceholderAPI instance that can be used to format text
+     */
+    public static PlaceholderAPI createInstance() {
+        return new PlaceholderAPI();
     }
```

Comment:
could this method be moved above `register`?

Reply:
Moved